### PR TITLE
automate 4 test cases

### PR DESCRIPTION
**Pull request by lihongyan1**
Related to PR
https://github.com/openshift/cucushift/pull/8202

### CODE REVIEW

**Comment by juzhao** [comment]
LGTM

File: lib/rules/cli/4.1.yaml [hunk 1]
```diff
@@ -834,6 +834,8 @@
     :image: --image=<value>
     :limits: --limits=<value>
     :oc_opt_end: --
+    :exec_command: <value>
```
Comment (pruan-rht) [inline lib/rules/cli/4.1.yaml:837]: @lihongyan1 please put the commands in alphabetical order

File: features/step_definitions/pod.rb [hunk 2]
```diff
@@ -269,3 +269,22 @@
   cb[cb_name] = system_pod_container_id[1].strip
 end
 
+Given /^I check containers cpu request for pod named #{QUOTED} under limit:$/ do |pod_name, table|
+    container_cpu_hash = table.rows_hash
```
Comment (pruan-rht) [inline features/step_definitions/pod.rb:273]: @lihongyan1 ruby indentation is 2-spaces instead of 4.

File: lib/rules/cli/4.5.yaml [hunk 3]
```diff
@@ -839,6 +839,8 @@
     :command: --command=<value>
     :dry_run: --dry-run
     :env: --env=<value>
+    :exec_command: <value>
+    :exec_command_arg: <value>
```
Comment (liangxia) [inline lib/rules/cli/4.5.yaml:843]: Checked with 4.6.0-202010271321.p0-ffd6836, I didn't see those options

Comment (liangxia) [reply lib/rules/cli/4.5.yaml:843]: Ack

**Pull request merged**